test/projection-lines.js

```diff
@@ -20,11 +20,12 @@ tape('glsl projection-line test', function (t) {
         'attribute vec2 p;',
         'void main () {',
         '  vec2 pixel = p;',
-        '  vec2 clip = 0.25 * (pixel - 3.5);',
+        '  vec2 clip = 0.25 * (pixel - 3.45);',
```

Comment:
A bit of a magic number, but I won't lose sleep over it.

Reply:
Yep, Travis has a different opinion about lines snapping in headless gl